src/components/formio/radio.tsx

```diff
@@ -51,9 +55,16 @@ export const Radio: React.FC<RadioProps> = ({
   options,
   label,
   required = false,
+  isClearable = true,
   tooltip = '',
   description = '',
 }) => {
+  const {values, setFieldValue} = useFormikContext<ExtendedComponentSchema>();
+
+  const CLEAR_VALUES_ICON = css`
+    margin-right: 5px;
+  `;
```

Comment:
this should be moved outside of the component implementation, it's a constant and doesn't need re-calculating on every render.

It will also cause merge conflicts with my #141  PR :(